C-Plus-Plus/README.md

```diff
@@ -230,3 +230,5 @@ _add list here_
 - [Sudoku Solver](other/sudoku.cpp)
 - [Swap Two Nibbles](other/SwapTwoNibbles.cpp)
 - [Unique_Number_III](other/Unique_Number_III.cpp)
+- [Count Set no. of bits](other/countsetbits.cpp)
+- [BrianKenighan's Algorithm to count set no. of bits](other/BrianKenighan'sAlgorithm.cpp)
```

Comment:
These changes make no sense?

Reply:
> These are added as all algorithms which are added are given like this only. That's why I added. If you tell I will remove it.

The changes in the README is not corresponding to the file you have added in this PR. You need to update readme for Sleep Sort only.